### PR TITLE
Review typos and links

### LINKED ISSUE #951: Review documentation

### PR DESCRIPTION
**Pull request by hnformentin**
**Issue**
Resolves #951

**Approach**
Review typos and links.

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -400,7 +400,7 @@ must place, in the relevant source files, a statement of the
 additional terms that apply to those files, or a notice indicating
 where to find the applicable terms.
 
-  Additional terms, permissive or non-permissive, may be stated in the
+  Additional terms, permissive or non-permissive, it may be stated in the
```
Comment (ManInFez) [inline COPYING:403]: I think the original was correct, as it refers to multiple terms.

File: docs/rst/manual/reference/configuration/data_types.rst [hunk 2]
```diff
@@ -569,7 +569,7 @@ situation:
 Some external Software (e.g. Cohiba) makes a large vector of random numbers
 which will serve as input to the forward model. (It is no requirement that the
 parameter set is large, but if it only consists of a few parameters the GEN_KW
-type will be easier to use.) We want to update this parameter with ERT. In
+type will be easier to use). We want to update this parameter with ERT. In
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/data_types.rst:572]: This does not need to be in a parenthesis

File: docs/rst/manual/reference/configuration/data_types.rst [hunk 3]
```diff
@@ -615,7 +615,7 @@ The template file can look like this (only the Magic123 is special):
 
 	Header line1
 	Header line2
-  ============
+   ============
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/data_types.rst:618]: should the line match the  line below?

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 4]
```diff
@@ -47,11 +47,11 @@ Keyword name                                                        	Required
 :ref:`ENKF_MERGE_OBSERVATIONS <enkf_merge_observations>`            	NO                    			FALSE                 		Should observations from many times be merged together
 :ref:`ENKF_NCOMP <enkf_ncomp>`                                        	NO                    			              			Number of PC to use when forcing a fixed number; used in combination with kw ENKF_FORCE_NCOMP 
 :ref:`ENKF_PEN_PRESS <enkf_pen_press>`                              	NO                    			FALSE                 		Should we want to use a penalised PRESS statistic in model selection? 
-:ref:`ENKF_RERUN <enkf_rerun>`                                      	NO                    			FALSE                 		Should the simulations be restarted from time zero after each update. 
+:ref:`ENKF_RERUN <enkf_rerun>`                                      	NO                    			FALSE                 		Should the simulations be restarted from time zero after each update?
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:50]: Should this be a question?

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 5]
```diff
@@ -47,11 +47,11 @@ Keyword name                                                        	Required
...
-:ref:`ENKF_RERUN <enkf_rerun>`                                      	NO                    			FALSE                 		Should the simulations be restarted from time zero after each update. 
+:ref:`ENKF_RERUN <enkf_rerun>`                                      	NO                    			FALSE                 		Should the simulations be restarted from time zero after each update? 
 :ref:`ENKF_SCALING <enkf_scaling>`                                  	NO                    			TRUE           		       	Deprecated.
 :ref:`ENKF_TRUNCATION <enkf_truncation>`                            	NO                    			0.99        	          	Cutoff used on singular value spectrum. 
 :ref:`ENSPATH <enspath>`                                            	NO                    			storage     	          	Folder used for storage of simulation results.
-:ref:`FIELD <field>`                                                	NO                                          				Ads grid parameters
+:ref:`FIELD <field>`                                                	NO                                          				Adds grid parameters
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:54]: Adds _a_ grid?

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 6]
```diff
@@ -180,14 +180,14 @@ These keywords must be set to make ERT function properly.
   
 		ECLBASE eclipse/model/MY_VERY_OWN_OIL_FIELD-%d
 
-	**Note that JOBNAME can be used as an alternative to ECLBASE.**
+	Note that JOBNAME can be used as an alternative to ECLBASE.
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:183]: remove highlight?

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 7]
```diff
@@ -1067,7 +1067,7 @@ and/or history matching project.
 	Some external Software (e.g. Cohiba) makes a large vector of random numbers
 	which will serve as input to the forward model. (It is no requirement that the
 	parameter set is large, but if it only consists of a few parameters the GEN_KW
-	type will be easier to use.) We want to update this parameter with ERT. In
+	type will be easier to use). We want to update this parameter with ERT. In
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:1070]: remove parenthesis

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 8]
```diff
@@ -1656,7 +1656,7 @@ but required when installing ERT at a new site.
 
 	Observe that when you refer to the keys later in the config file they must be
 	enclosed in '<' and '>'. Furthermore, a key-value pair must be defined in the
-	config file before it can be used. The final key define above KEY, will be
+	config file before it can be used. The last key define above (KEY) will be
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:1659]: should this be: "The last key defined above..."?

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 9]
```diff
@@ -1849,13 +1849,13 @@ Available LSF configuration options
 .. topic:: LSF_QUEUE
 
 	The name of the LSF queue you are running simulations in.
-	For e.g. ``bsub``, this option will be passed to the ``-q`` parameter:
-	https://www.ibm.com/support/knowledgecenter/en/SSETD4_9.1.3/lsf_command_ref/bsub.q.1.html
-
+	For example, ``bsub``, this option will be passed to the ``-q`` parameter:
+	https://www.ibm.com/support/knowledgecenter/SSWRJV_10.1.0/lsf_command_ref/bsub.q.1.html
+
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:1854]: remove whitespace

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 10]
```diff
@@ -2236,7 +2236,7 @@ instance, and are not applied to the shell.
 .. topic:: UPDATE_PATH
 
 	The UPDATE_PATH keyword will prepend a new element to an existing PATH
-	variable. I.e. the config
+	variable, i.e. the config
```
Comment (ManInFez) [inline docs/rst/manual/reference/configuration/keywords.rst:2239]: missing .

File: ert_gui/resources/gui/help/config/simulation/target_case.html [hunk 11]
```diff
@@ -1 +1 @@
-This is the name of the case where the results for the updated parameters will be stored
\ No newline at end of file
+This is the name of the case where the results for the updated parameters will be stored.
```
Comment (ManInFez) [inline ert_gui/resources/gui/help/config/simulation/target_case.html:1]: no newline at end of file

File: ert_shared/main.py [hunk 12]
```diff
@@ -115,7 +115,7 @@ def get_ert_parser(parser=None):
     )
     subparsers.required = True
 
-    config_help = "Ert configuration file"
+    config_help = "Ert configuration file."
```
Comment (ManInFez) [inline ert_shared/main.py:118]: capitalize ERT

Comment (ManInFez) [reply ert_shared/main.py:118]: Not sure if I agree on punctuation in the help

**Comment by hnformentin** [comment]
Thanks for the comments :-D

File: docs/rst/manual/reference/configuration/keywords.rst (hunk 9 above)
Comment (jondequinor) [reply docs/rst/manual/reference/configuration/keywords.rst]: I think this one was actually meant to be "For bsub, this option […]". There's no example here.

**Pull request merged**